sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/MethodShouldBeNameAccordingToSynchronicityTest.cs

```diff
@@ -40,6 +40,34 @@ public void MethodShouldBeNameAccordingToSynchronicity(string tasksVersion)
                 additionalReferences: NuGetMetadataReference.SystemThreadingTasksExtensions(tasksVersion));
         }
 
+        [TestMethod]
+        [DataRow("3.0.20105.1")]
+        [DataRow(Constants.NuGetLatestVersion)]
+        [TestCategory("Rule")]
+        public void MethodShouldBeNameAccordingToSynchronicity_MVC(string mvcVersion)

```

Comment:
```suggestion
        public void MethodShouldBeNamedAccordingToSynchronicity_MVC(string mvcVersion)
```
(and apply the name change everywhere...)